xml/http_servers.xml

```diff
@@ -1907,6 +1907,27 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
+   <fingerprint pattern="^BBC (\d+\.\d+\.\d+); ovbbccb \d+\.\d+\.\d+$">
+      <example>BBC 06.00.083; ovbbccb 06.00.083</example>
+      <example>BBC 06.20.050; ovbbccb 06.20.050</example>
+      <example>BBC 11.10.035; ovbbccb 11.10.035</example>
+      <description>OpenView ovbbccb </description>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.product" value="OpenView ovbbccb"/>
```

Comment:
Is it correct and important to be calling this ovbbccb?  I have a similar change that I haven't put up for a PR yet that looks like:

```
+   <!-- fingerprints various services exposed by HP OpenView components.  For
+        now, the versions and components are not extracted because it is
+        unclear if these are relevant/accurate/useful.
+
+        BBC == Black Box Communications
+        CB == Communications Broker
+        CODA == COmmunications DAemon
+        LLB == Local Location Broker
+        RCP == Reverse Channel Proxy
+   -->
+   <fingerprint pattern = "^BBC (?:[\d\.]+); (?:\S+) (?:[\d\.]+|unknown version)$">
+      <description>HP Openview</description>
+      <example>BBC 06.20.101; ovbbccb 06.20.101</example>
+      <example>BBC 11.13.007; ovbbccb unknown version</example>
+      <example>BBC 05.20.050; coda 10.00.123</example>
+      <example>BBC 2.6.0.7; com.hp.openview.Coda 0.0.1</example>
+      <example>BBC 2.6.0.7; com.hp.openview.bbc.LLBServer 2.6.0.7</example>
+      <example>BBC 11.00.044; ovbbcrcp 11.00.044</example>
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.family" value="OpenView"/>
+      <param pos="0" name="service.product" value="OpenView"/>
    </fingerprint>
```

If splitting it out by the different part makes sense (based on what you did here, it does), then should we also extract the product specific version?  IOW, there are two versions in this string which may not be the same -- is one the OpenView version and the other the specific component's version?